tests/components/google/test_init.py

```diff
@@ -699,7 +699,10 @@ async def test_add_event_location(
 
 @pytest.mark.parametrize(
     "config_entry_token_expiry",
-    [datetime.datetime.max.replace(tzinfo=UTC).timestamp() + 1],
+    [
+        (datetime.datetime.max.replace(tzinfo=UTC).timestamp() + 1),
+        (utcnow().replace(tzinfo=None).timestamp()),
+    ],
```

Comment:
I think you need something like this to fix CI so that the test ids are stable

```suggestion
    ],
    ids=["max_timestamp","timestamp_naive"],
```